Description: Remove debug strip from make
 Because "strip" is done by debhelper already,
 the removal of debugging symbols should be removed
 from the upstream Makefile.
Author: Aditya Vaidya <kroq.gar78@gmail.com>
Forwarded: no
Last-Update: 2012-08-25
diff --git a/Makefile b/Makefile
index b2d4570..6246fc6 100644
--- a/Makefile
+++ b/Makefile
@@ -75,7 +75,7 @@ build/obj/%.powder-sse3.o: src/%.c $(HEADERS)
 
 build/powder-sse2: $(patsubst build/obj/%.o,build/obj/%.powder-sse2.o,$(OBJS))
 	$(CC) $(CFLAGS) $(LDFLAGS) $(EXTRA_OBJS) $(patsubst build/obj/%.o,build/obj/%.powder-sse2.o,$(OBJS)) $(LIBS) -o $@
-	strip $@
+	#strip $@
 build/obj/%.powder-sse2.o: src/%.c $(HEADERS)
 	$(CC) -c $(CFLAGS) -o $@ $<
 
@@ -100,7 +100,7 @@ build/obj/%.powder-64-sse3.o: src/%.c $(HEADERS)
 
 build/powder-64-sse2: $(patsubst build/obj/%.o,build/obj/%.powder-64-sse2.o,$(OBJS))
 	$(CC) $(CFLAGS) $(LDFLAGS) $(EXTRA_OBJS) $(patsubst build/obj/%.o,build/obj/%.powder-64-sse2.o,$(OBJS)) $(LIBS) -o $@
-	strip $@
+	#strip $@
 build/obj/%.powder-64-sse2.o: src/%.c $(HEADERS)
 	$(CC) -c $(CFLAGS) -o $@ $<
 
